components/Service/ServicePage.js

```diff
@@ -0,0 +1,172 @@
+/* eslint-disable react/jsx-indent-props */
+import React from 'react';
+import Head from 'next/head';
+import NavigationBar from '../Common/NavigationBar';
+import './service.scss';
+import { HOME_STRINGS } from '../constants';
+import { SERVICE_STRING } from './constants';
```

Comment:
separation of concerns is important. move strings for this component to same place

Reply:
Like you said I should practice DRY so I can't be repeating the same string that already exists when I can just import it from the initial place and if you pay enough attention to details you'll notice that those two constants are serving two very different purpose